### PR TITLE
This task will complete task 5 and the respective tests

### PR DESCRIPTION
**Pull request by AndreiThira**

### CODE REVIEW

**Review by mezzDavies — CHANGES_REQUESTED** [review]
Good happy path testing 👍  but there is nothing here to test for or handle errors for this endpoint. Please add this sad path and resubmit.  

Also, starting from your next task/branch please name the branches after the task number (this is task 6 not 5 and it's very confusing this side of things 😄 )

(Oh, finally, don't forget to update your endpoints.json!)

File: models/topic-model.js [hunk 1]
```diff
@@ -43,11 +43,16 @@ ORDER BY
   })
 }
 
-const fetchCommentsByArticle = (article_id) =>{
-  return db.query("SELECT * FROM COMMENTS WHERE article_id = $1 ORDER BY created_at DESC;", [article_id]).then(({rows})=>{
-    return rows;
-  })
-}
+const fetchCommentsByArticle = async (article_id) => {
```
Comment (nc-poppy-m) [inline models/topic-model.js]: It's good to stay consistent, either with ```async/await``` or ```.then```  throughout, so I'd encourage you to keep one or the other for the sake of consistency in your portfolio piece :)

**Review by nc-poppy-m — CHANGES_REQUESTED** [review]
Great work implementing feedback! Only one issue - you should be consistently using either async/await or promises, so make sure your```fetchAllArticles``` and ```fetchCommentsByArticle``` are handled in the same way. Fix that up and resubmit this pull request tomorrow 🙂

File: models/topic-model.js [hunk 2]
```diff
@@ -21,38 +21,50 @@ const fetchArticleByID = (articleID)=>{
...
 
-const fetchCommentsByArticle = async (article_id) => {
-  const articleExists = await db.query("SELECT COUNT(*) FROM articles WHERE article_id = $1;", [article_id]);
 
-  if (articleExists.rows[0].count === '0') {
-    return Promise.reject({status:404, message: "Article not found"})
-  }
+const fetchCommentsByArticle = (article_id) => {
+  return db.query("SELECT COUNT(*) FROM articles WHERE article_id = $1;", [article_id])
+    .then(articleExistsResult => {
```
Comment (mezzDavies) [inline models/topic-model.js:53]: I would have a look at [these notes](https://notes.northcoders.com/courses/js-back-end/advanced-error-handling) about how to implement this checkExists and make it reusable 👍

**Review by mezzDavies — APPROVED** [review]
Approved but I have left a comment as a strong recommendation 👍 

(Don't forget to name the branch after task next time round! 😄 )

**Pull request merged**